pkg/controller/networkpolicy/networkpolicy.go

```diff
@@ -252,6 +252,15 @@ func ForService(req router.Request, resp router.Response) error {
 			Protocol: &proto,
 			Port:     &targetPort,
 		})
+
+		// Healthchecks in AWS that are performed on LoadBalancer Services use TCP even if the port is UDP,
```

Comment:
Will that work in other envs other than aws?

Reply:
It certainly shouldn't break anything on other envs. We're gonna move the NetworkPolicy code into Manager at some point anyway and we can customize everything more once it's there.